utils/src/unit-test/java/uk/gov/moj/sdt/utils/logging/PerformanceLoggerTest.java

```diff
@@ -70,7 +72,7 @@ protected void setUpLocalTests() throws Exception {
                 outChan.truncate(0);
                 outChan.close();
             } catch (final Exception e) {
-                Assert.fail(e.getStackTrace().toString());
+                fail(Arrays.toString(e.getStackTrace()));

```

Comment:
can we just call fail, do we need ot convert stackstrace to String?

Reply:
Legacy code here. What exactly would you like to have as the replacement?